qa-tests-backend/src/main/groovy/util/Env.groovy

```diff
@@ -30,6 +30,10 @@ class Env {
     // the remote cluster arch can be ppc64le or s390x, default is x86_64
     static final REMOTE_CLUSTER_ARCH = System.getenv("REMOTE_CLUSTER_ARCH") ?: "x86_64"
 
+    // STANDALONE_SECURED_CLUSTER specifies that the remote cluster being used to execute tests
+    // only has secured-cluster deployed and connects to a remote central
+    static final STANDALONE_SECURED_CLUSTER = System.getenv("STANDALONE_SECURED_CLUSTER") ?: "true"
```

Comment:
I think make this ONLY_SECURED_CLUSTER and default to "false"?
```suggestion
    // ONLY_SECURED_CLUSTER specifies that the remote cluster being used to execute tests
    // only has secured-cluster deployed and connects to a remote central
    static final ONLY_SECURED_CLUSTER = System.getenv("ONLY_SECURED_CLUSTER") ?: "false"
```

Reply:
It will run the Cert* tests unless ONLY_SECURED_CLUSTER is explicitly set to "true". Which is what I thought the intent of this PR was i.e. to have a way to skip these tests when Central is not deployed to "remote".
```
@IgnoreIf({ Env.ONLY_SECURED_CLUSTER == "true" })
```

Reply:
Ah yes! my bad. false means the test should be executed. So CI will execute test by default. 
On P/Z we need to set it to true based on where central is deployed.

Reply:
Yeah, it is confusing. Maybe there is a better way to word it. I checked that the tests still run in the existing e2e-tests and so will approve.